packages/cli/src/start.ts

```diff
@@ -92,6 +92,19 @@ export async function rnxStart(
     cliOptions.customLogReporterPath
   );
 
+  const printHelp = () => {
+    const dim = chalk.dim;
+    const press = dim(" › Press ");
+    [
+      ["r", "reload the app"],
+      ["d", "open developer menu"],
+      ["a", "show bundler address QR code"],
+      ["ctrl-c", "quit"],
```

Comment:
I think we should also add `h` to the help message:

```suggestion
      ["a", "show bundler address QR code"],
      ["h", "show this help message"],
      ["ctrl-c", "quit"],
```